src/app/components/canvas/canvas.component.ts

```diff
@@ -121,8 +124,10 @@ export class CanvasComponent {
     this.postsService.getAll(this.boardID).then((data) => {
       data.forEach((data) => {
         let post = data.data() ?? {}
-        let obj = JSON.parse(post.fabricObject);
-        this.syncBoard(obj, post.postID);
+        if(post.fabricObject){
```

Comment:
post.fabricObject will be false if fabricObject is "" since "" is falsy. I can change it to null
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
TIL!